client/components/Chat.vue

```diff
@@ -58,8 +58,10 @@
 					v-else
 					class="chat-content">
 					<div
-						:class="['scroll-down', {'scroll-down-shown': !channel.scrolledToBottom}]"
-						@click="$refs.messageList.jumpToBottom()">
+						:class="['scroll-down tooltipped tooltipped-w', {'scroll-down-shown': !channel.scrolledToBottom}]"
+						aria-label="Jump to recent messages"
```

Comment:
I started to "Scroll to bottom" which I didn't like (too cold/formal, also not a "scroll" per se) then shamelessly stole the wording from Slack, which I found more accurate and friendly.